nixos/modules/services/databases/postgresql.nix

```diff
@@ -168,6 +168,18 @@ in
                 }
               '';
             };
+            ensureRoles = mkOption {
```

Comment:
_Roles_ and _Users_ are synonymous in PostgreSQL.

The linked page calls them options. [This page](https://www.postgresql.org/docs/15/sql-createrole.html) calls them clauses. That seems like an even better name.